### DISCUSSION TITLE
Auto equipping items

### DISCUSSION THREAD

**Discussion by cable-one** [isid0re/SoloLeveling#84] [Q&A]
Hi,

Thanks for your great scripts!

Iam a total noob, but with the guide provided i got it up running in no time.

For now iam trying to make a wind druid.
The leveling is all fine.
It just finished normal.

Now it is starting nightmare and ive decided to check its gear.
It seems that my merc is getting better equipped then me.
I am still running around in blue items wich require lvl 15 tops, while i am lvl 47 already.

Is this correct?
There are no items in my inventory, and i havent changes anything in the pickit.

**Comment by isid0re** [comment]
Druid is a caster. So its focus on gear is different than the mercs or a melee character. Just let it run as gear is dependent on RNG. Also, from normal to nightmare the gear itself is never really spectacular.
